.github/workflows/e2e.yaml

```diff
@@ -161,7 +161,7 @@ jobs:
         uses: engineerd/setup-kind@v0.5.0
         with:
           version: "v0.20.0"
-          image: kindest/node:v1.27.3
+          image: kindest/node:v1.28.0@sha256:b7a4cad12c197af3ba43202d3efe03246b3f0793f162afb40a33c923952d5b31
```

Comment:
why use a sha here?

Reply:
from the kind [releases page](https://github.com/kubernetes-sigs/kind/releases/tag/v0.20.0)

> Additional images built for this release:
    1.28: kindest/node:v1.28.0@sha256:b7a4cad12c197af3ba43202d3efe03246b3f0793f162afb40a33c923952d5b31
    

>   NOTE: You must use the @sha256 digest to guarantee an image built for this release, until such a time as we switch to a different tagging scheme. Even then we will highly encourage digest pinning for security and reproducibility reasons.

shall I give it a try without the sha once?